client/components/UI/Form/TimeInput/index.tsx

```diff
@@ -100,14 +117,14 @@ export class TimeInput extends React.Component {
                 invalid: true,
                 viewValue: val,
                 showLocalValidation: false,
-            });
+            }, this.updateInputWidth);
```

Comment:
Do not add `this.updateInputWidth` call here. Move to `componentDidUpdate` instead.